mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown

```diff
@@ -113,3 +113,17 @@ Description of the change and how users should adjust their configuration (if ne
 ### `settings.ip_configuration.require_ssl` is now removed
 
 Removed in favor of field `settings.ip_configuration.ssl_mode`.
+
+## Resource: `google_vpc_access_connector`
+
+### Default values removed for `min_throughput` and `max_throughput`
+
+The fields `min_throughput` and `max_throughput` no longer have default values set by the provider. This was necessary to add conflicting field validation. described in the next section.
+
+No configuration changes are needed for existing resources as these fields' values will default to values present in data returned from the API.
+
+### Conflicting field validation add for `min_throughput` and `min_instances`, and `min_throughput` and `max_throughput`
```

Comment:
Nit: "added"

Also, should the ending be "and `max_throughput` and `max_instances`"?

Reply:
Yes, thank you! 🤦  I was a little too copy/paste happy here